service/backend/pchain/account.go

```diff
@@ -417,7 +417,7 @@ func (b *Backend) fetchUTXOsAndStakedOutputs(ctx context.Context, addr ids.Short
 	var stakedUTXOBytes [][]byte
 	if fetchStaked {
 		// fetch staked outputs for addr
-		_, stakedUTXOBytes, err = b.pClient.GetStake(ctx, []ids.ShortID{addr})
+		_, stakedUTXOBytes, err = b.pClient.GetStake(ctx, []ids.ShortID{addr}, true)
```

Comment:
@bilgehansahin I'd add a comment here stating that we assume staking is done via validation, not delegation. Just to help us recall it should this change in the future.